library/src/scripts/styles/typographyStyles.ts

```diff
@@ -23,18 +24,25 @@ export const typographyClasses = useThemeCache(() => {
         {
             fontSize: unit(globalVars.fonts.size.title),
             lineHeight: globalVars.lineHeights.condensed,
-            $nest: lineHeightAdjustment(globalVars.lineHeights.condensed, true),
-        },
+            transform: `translateX(${em(globalVars.fonts.alignment.headings.horizontal)})`,
+            $nest: lineHeightAdjustment() as NestedCSSSelectors,
```

Comment:
I have the same note here about the `NestedCSSSelectors` typing. This would allow us to remove the imports and remove both `as` declarations here.